portfolio/src/main/webapp/script.js

```diff
@@ -42,46 +42,46 @@ function defaultPage() {
  * Fetches messages from the servers and adds them to the DOM.
  */
 async function getComments() {
-    fetch('/data').then(response => response.json()).then((msgs) => {
+  fetch('/data').then(response => response.json()).then((msgs) => {
    
-    const statsListElement = document.getElementById('comments-container');
-    statsListElement.innerHTML = '';
-    msgs.forEach((msg) => {
-        statsListElement.appendChild(
-            createListElement(msg.nickname + ': ' + msg.message));
-        statsListElement.appendChild(
-            createImgElement(msg.image));
+  const statsListElement = document.getElementById('comments-container');
+  statsListElement.innerHTML = '';
+  msgs.forEach((msg) => {
+    statsListElement.appendChild(
+      createListElement(msg.nickname + ': ' + msg.message));
+    statsListElement.appendChild(
+      createImgElement(msg.image));
     })
     
   });
 }
 
 function getCommentsHome() {
-    fetch('/data').then(response => response.json()).then((msgs) => {
+  fetch('/data').then(response => response.json()).then((msgs) => {
    
-    const statsListElement = document.getElementById('home-comments-container');
-    statsListElement.innerHTML = '';
-    msgs.forEach((msg) => {
-        statsListElement.appendChild(
-            createListElement(msg.nickname + ': ' + msg.message));
-        statsListElement.appendChild(
-            createImgElement(msg.image));
+  const statsListElement = document.getElementById('home-comments-container');
+  statsListElement.innerHTML = '';
+  msgs.forEach((msg) => {
+    statsListElement.appendChild(
+      createListElement(msg.nickname + ': ' + msg.message));
+    statsListElement.appendChild(
+      createImgElement(msg.image));
```

Comment:
Indentation still looks a bit off here. Please refer to this page[1] got more official/detailed info for indentation in JS, particularly, " Indent continuation lines at least +4 spaces".

[1] https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks